helm/cloudnative-pg/Chart.yaml

```diff
@@ -12,4 +12,7 @@ annotations:
   config.giantswarm.io/version: 1.x.x
 dependencies:
   - name: cloudnative-pg
-    version: 0.21.5
+    version: 0.23.2
+  - name: cluster
+    version: 0.0.4
+    repository: "https://cloudnative-pg.github.io/grafana-dashboards"
```

Comment:
The chart in this repo was broken previously but this makes me think it has been fixed -- did vendir do the chart update? If so, :tada:

Reply:
No I did it by hand ;)

Reply:
I'll release it tomorrow ;)